src/cljc/rems/common/catalogue_util.cljc

```diff
@@ -1,9 +1,33 @@
 (ns rems.common.catalogue-util
-  (:require [clojure.string :as str]))
+  (:require [clojure.string :as str]
+            [rems.text :refer [localized]]))
 
-(defn urn? [resid]
+(defn- urn? [resid]
   (and resid (str/starts-with? resid "urn:")))
 
-(defn urn-catalogue-item-link [{:keys [resid]} {:keys [urn-organization]}]
+(defn- urn-catalogue-item-url [resid {:keys [urn-organization]}]
   (when (urn? resid)
     (str (or urn-organization "http://urn.fi/") resid)))
+
```

Comment:
the EGA url stuff could be split out to a separate PR and just merged

Reply:
https://github.com/CSCfi/rems/pull/2605